engines/glk/zcode/detection_tables.h

```diff
@@ -3858,7 +3858,7 @@ const FrotzGameDescription FROTZ_GAMES[] = {
 	ENTRY0("awitl", "980215", "8cf72be55d75137cc435ab25f479c123", 56320),
 	ENTRY0("ayac", "010107", "b503a9f2817ce97480160e454e470c58", 68608),
 	ENTRY0("azteca", "070117", "e30320818aa935f1bff66fcaa862fd9f", 123904),
-	ENTRY0("b2demo", "961017/Demo", "28d166262aa22fb55fdb9685a5a124ef", 65024),
+	ENTRY0("b2demo", "961017", "28d166262aa22fb55fdb9685a5a124ef", 65024),
```

Comment:
and change this here
`ENTRY0("phoneboothp2", "961017/Demo", "28d166262aa22fb55fdb9685a5a124ef", 65024),`

Same thing for the other demos

Reply:
https://github.com/scummvm/scummvm/pull/5388/commits/b2741f5dcbb27d95df4288e2d3e594b40d5780d8